Makefile

```diff
@@ -41,13 +41,16 @@ help:
 ##@ Dependencies
 
 .PHONY: build-dev-deps
-build-dev-deps: ## Install dependencies for builds
+build-dev-deps: ## Install dependencies for development
```

Comment:
Should I rename this target to `install-dev-deps` for matching with the new added target?